pybossa/view/projects.py

```diff
@@ -3021,7 +3021,7 @@ def answerfieldsconfig(short_name):
     project_sanitized, owner_sanitized = sanitize_project_owner(
         project, owner, current_user, ps)
     answer_fields = project.info.get('answer_fields', {})
-    consensus_config = project.info.get('tie_break_config', {})
+    consensus_config = project.info.get('consensus_config', {})
```

Comment:
Can you reference the variable `key` here (if it is in scope), and not need to use the hard-coded string `'consensus_config'`?

Reply:
yes you are right, we should add a key variable instead of hardcode